doc/testing_policy.md

```diff
@@ -0,0 +1,56 @@
+# Testing policy
+## Contents
+1. [Introduction](#1-introduction)
+2. [How to start Test Suite (Local)](#2-how-to-start-test-suite-local)  
+    2.1 [Using the system build script](#21-using-the-system-build-script)  
+    2.2 [Using standard Go language facilities](#22-using-standard-go-language-facilities)  
+3. [Automated Run Test Suite (Remote)](#3-automated-run-test-suite-remote)  
+
+---
+
+## 1. Introduction
+
+Testing is a very important part of Edge Orchestration project and our team values it highly.
+
+**When adding or changing functionality, the main requirement is to include new tests as part of your contribution.**
```

Comment:
What about soothing the requirement? This is due to my personal concern on pushing a pressure on those who wants to contribute their code to our community, since we are still at the early stage and needs more contributions from 3rd party developers.

Reply:
In this case, I suggest an addition:
> If your contribution does not have the required tests, please mark this in the PR and our team will develop them.

Reply:
I'd like to revise `our team will develop them.` to `our team will support to develop them.` 😁

Reply:
Ok